packages/contracts/deploy/40_finalize-managing-dao/30_install-multisig-on-managing-dao.ts

```diff
@@ -55,29 +57,37 @@ const func: DeployFunction = async function (hre: EHRE) {
 
   // Install multisig build 2
   const multisigRepoAddress = hre.aragonPluginRepos['multisig'];
-  const versionTag = [1, 2];
-  const pluginSetupRef = [versionTag, multisigRepoAddress];
+  const versionTag = {
```

Comment:
Maybe we can adapt the 
`export type VersionTag = [number, number];` accordingly in `packages/contracts/test/test-utils/psp/types.ts` and use it here as well.

Reply:
I would rather not. `packages/contracts/test/test-utils/psp/types.ts` is used by tests and is not shared with the deployment scripts